src/Object/Optional.ts

```diff
@@ -42,3 +42,16 @@ export type Optional<O extends object, K extends Key = Key, depth extends Depth
     0: MergeFlat<OptionalPart<Pick<O, K>, depth>, O>
     // Pick a part of O (with K) -> nullable -> merge it with O
 }[Implements<Keys<O>, K>] & {}
+
+/**
+Partial is an alias for Optional with a simpler syntax. Just like the native `Partial` it makes a type's fields optional, but accepts one extra param to choose between 'flat' or 'deep'.
+@param O the type to make its fields optional
+@param depth (?=`'flat'`) to do it deep
+@returns [[Object]]
+@example Partial<MyType>
+@example Partial<MyType, 'deep'>
+ */
+export type Partial<O extends object, depth extends Depth = 'flat'> = {
```

Comment:
I'm ok with this, but we should put this inside another file called `Partial.ts`